mantaray/commands.py

```diff
@@ -89,6 +89,8 @@ def _define_commands() -> dict[str, Callable[..., None]]:
     def join(view: View, core: IrcCore, channel: str) -> None:
         # TODO: plain '/join' for joining the current channel after kick?
         core.send(f"JOIN {channel}")
+        if "away-notify" in core.cap_list:
+            core.send(f"WHO {channel}")
```

Comment:
This will send `WHO #css`, if you send `JOIN #css` but the server redirects you to `#css-overflow`. We should probably send `WHO` in `_handle_endofnames()`.